testing/file_test/file_test_base.cpp

```diff
@@ -316,16 +324,36 @@ auto FileTestBase::ProcessTestFileAndRun(TestContext& context)
   llvm::PrettyStackTraceProgram stack_trace_entry(
       test_argv_for_stack_trace.size() - 1, test_argv_for_stack_trace.data());
 
+  // Conditionally capture console output. We use a scope exit to ensure the
+  // captures terminate even on run failures.
+  std::unique_ptr<std::unique_lock<std::mutex>> console_lock;
```

Comment:
Would it work to use `unique_lock` directly, without the `unique_ptr` wrapper? `unique_lock` is default-constructible and move-assignable, which seems like all you need here.

Reply:
Done, missed that.